backend/src/zac/camunda/api/views.py

```diff
@@ -66,6 +66,7 @@ def get(self, request: Request, *args, **kwargs):
         zaak_url = request.GET.get("zaak_url")
         if not zaak_url:
             err_serializer = ErrorSerializer(data={"detail": "missing zaak_url"})
+            err_serializer.is_valid()
```

Comment:
```suggestion
            err_serializer = ErrorSerializer({"detail": "missing zaak_url"})
```

I don't quite remember why the serializer is needed here, since the data is always the same. But I think that validating the known data is a bit excessive, but I'll live it up to you.